docs/USING_PUBLIC_CLOUDS.rst

```diff
@@ -394,3 +394,113 @@ endpoint :code:`zenkotos3.com`:
     },
     (...)
 
+    Backblaze as a data backend
+    ---------------------------
+
+    From the Backblaze b2 web page
+    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+    From the Backblaze b2 web page, from your Storage Account dashboard, create a bucket where you will host your data for this new location constraint.
+
+    You will also need to get one of your b2 application key and to provide it to CloudServer. This can be found from B2 Cloud Storage Buckets dashboard: click on 'Show Account ID and Application Key', then select you account ID and click on 'Create Application Key'.
+    Important: When you create a new application key, the old one will no longer work.
+
+    In this example, our bucket will be named ``b2bucket``.
```

Comment:
Consistency, trivial: why have two backquotes here, when you also use only one in many places of your doc ?